index.html

```diff
@@ -0,0 +1,103 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <style>
+        * {
+            box-sizing: border-box;
+        }
+
+        body {
+            font-family: Arial, Helvetica, sans-serif;
+            background-color: #666
+        }
+
+        /* Style the header */
+
+        header {
+            background-color: #666;
+            padding: 5px;
+            text-align: center;
+            font-size: 20px;
+            color: white;
+        }
+
+        nav {
+            float: left;
+            width: 30%;
+            height: 400px;
+            background: #666;
+            padding: 20px;
+        }
+
+            /* Style the list inside the menu */
+            nav ul {

```

Comment:
pay attention to indentation. This block should be moved 4 spaces left to be aligned with the rest.